versions.yaml

```
@@ -3,7 +3,7 @@ SCCACHE_VER: 0.7.7
 # renovate: datasource=github-releases depName=cli/cli
 GH_CLI_VER: 2.51.0
 # renovate: datasource=github-releases depName=codecov/uploader
-CODECOV_VER: 0.7.2
+CODECOV_VER: 0.6.0
```

There is a new release: https://pypi.org/project/codecov-cli/#history
```suggestion
CODECOV_VER: 0.7.2
```

---

Latest version [here](https://pypi.org/project/codecov-cli/) reads `0.7.1`